### PR TITLE
Throttle DiscoVPC calls to avoid AWS rate limited exceptions

### PR DESCRIPTION
**Pull request by aoskotsky-amplify**

### CODE REVIEW

**Comment by jconstance-amplify** [comment]
It looks like this has worked well on rpt-jenkins, so 👍

File: disco_aws_automation/disco_vpc.py [hunk 1]
```diff
@@ -441,28 +444,33 @@ def _destroy_rds(self, wait=True):
...
-        def _destroy():
-            for interface in self.boto3_ec2.describe_network_interfaces(
-                    Filters=self.vpc_filters())["NetworkInterfaces"]:
 
-                self.boto3_ec2.delete_network_interface(NetworkInterfaceId=interface['NetworkInterfaceId'])
+        def _destroy():
+            interfaces = throttled_call(self.boto3_ec2.describe_network_interfaces,
+                                        Filters=self.vpc_filters())["NetworkInterfaces"]
+            for interface in interfaces:
+                interfaces(self.boto3_ec2.delete_network_interface,
```
Comment (jichen-amplify) [inline disco_aws_automation/disco_vpc.py]: replace `interfaces` with `throttled_call`?

Comment (aoskotsky-amplify) [inline disco_aws_automation/disco_vpc.py]: Oh wow. Don't know how I missed that

**Comment by kelibezhani** [comment]
Has this been tested with testbox VPC? Otherwise LGTM 👍

**Comment by IlyaSukhanov** [comment]
+1

although I do prefer
```
foobarbaz(
    really_long_bar,
)
```
over 
```
foobarbaz(
          really_long_bar,
)
```

**Comment by jichen-amplify** [comment]
👍

**Pull request merged**